pyqtgraph/parametertree/parameterTypes/pen.py

```diff
@@ -185,7 +185,14 @@ def _makeChildren(self, boundPen=None):
                 if p.type() != 'color':
                     p.sigValueChanging.connect(newSetter)
                 # Force children to emulate self's value instead of being part of a tree like normal
-                p.sigValueChanged.disconnect(p._emitValueChanged)
+                try:
+                    p.sigValueChanged.disconnect(p._emitValueChanged)
+                except RuntimeError:
+                    # workaround https://bugreports.qt.io/projects/PYSIDE/issues/PYSIDE-2487
+                    # that affects PySide 6.5.3
+                    # Since the child param was freshly created by us, there can only have been one slot
```

Comment:
Is it possible to guarantee this with `assert p.receivers("_emitValueChanged(object, object)") == 1`?

I tried various adaptations of arguments to `receivers` according to the [docs](https://doc.qt.io/qtforpython-6/PySide6/QtCore/QObject.html#PySide6.QtCore.PySide6.QtCore.QObject.receivers) example, but it always returns 0 when I try to test with a MWE. Do you know whether it works in Python? There aren't too many examples online...

Reply:
For PySide, the syntax is: `assert p.receivers(QtCore.SIGNAL("sigValueChanged(PyObject,PyObject)")) == 1`
The PySide syntax specific things are:
1) PyObject
2) QtCore.SIGNAL: PyQt has dropped it since moving to the new signal / slot syntax